shared/reducers/chat2.js

```diff
@@ -107,6 +107,9 @@ const metaMapReducer = (metaMap, action) => {
     }
     case Chat2Gen.metasReceived:
       return metaMap.withMutations(map => {
+        if (action.payload.clearExistingMetas) {
+          map.clear()
```

Comment:
you want to return here right?

Reply:
No, I just want to clear out the map before we start inserting. We still have a bunch of new metas that we want to insert into the metamap